kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/Reflector.java

```diff
@@ -72,11 +72,14 @@ private synchronized void stopWatcher() {
    * <br>Should be called only at start and when HttpGone is seen.
    */
   public void listSyncAndWatch() {
+    log.debug("Listing items for resource {}", apiTypeClass);
     running = true;
     KubernetesResourceList<T> result = null;
     String continueVal = null;
     Set<String> nextKeys = new LinkedHashSet<>();
+    long listStartTimeMillis = System.currentTimeMillis();
```

Comment:
It didn't make sense to pull in the guava dep for Stopwatch, so doing this the old fashioned way

Reply:
when timing things its better to use `System.nanoTime` b/c it guarantees monotonicity.